src/components/Main/Main.jsx

```diff
@@ -22,6 +22,7 @@ export const Main = () => {
     timeCreated: `${date.getDate()} - 
       ${date.getMonth() + 1} -
```

Comment:
for months you are not providing zeros, so I think for double digits it's okay, but for single digits it might look odd.
**Just a suggestion**, in case you wanna implement it , here is a good way

[https://bobbyhadz.com/blog/javascript-get-month-and-date-in-2-digit-format](url)

Reply:
Thanks for the suggestion I will definitely look onto it.